### PR TITLE
Release 0.4.0

### PR DESCRIPTION
**Pull request by jenniferboedker**
0.4.0 (2021-09-10)
------------------

**Added**

* List failing QC samples for single projects (<https://github.com/qbicsoftware/sample-tracking-status-overview/pull/77>)

* Count of samples that finished library prep is shown (<https://github.com/qbicsoftware/sample-tracking-status-overview/pull/89>)

* Enable user to subscribe to individual projects (<https://github.com/qbicsoftware/sample-tracking-status-overview/pull/84>)

**Fixed**

* Remove possibility of duplicate subscription of user per project (<https://github.com/qbicsoftware/sample-tracking-status-overview/pull/95>)

**Dependencies**

**Deprecated**

### CODE REVIEW

**Comment by jenniferboedker** [comment]
[Prepare release of sample-tracking-status-overview](https://trello.com/c/c8NIWF8o/979-prepare-release-of-sample-tracking-status-overview)

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,6 +5,26 @@ Changelog
 This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 
+0.4.0 (2021-09-10)
```
Comment (Steffengreiner) [inline CHANGELOG.rst:8]: NIT: 
```suggestion
0.4.0 (2021-09-13)
```

**Review by Steffengreiner — DISMISSED** [review]
LGTM 🚀

**Review by Steffengreiner — APPROVED** [review]
LGTM :rocket

**Pull request merged**